apps/extension/src/features/accounts/account-card.tsx

```diff
@@ -0,0 +1,376 @@
+                      sideOffset={10}
+                      className="z-10"
+                    >
+                      <div className="px-5 py-6 bg-white rounded-lg shadow-2xl w-[350]">
+                        <div className="font-semibold">Organizer Account</div>
+                        <div className="pt-2">
+                          The selected account will be the organizer for all
+                          events created.
+                        </div>
+                      </div>
+                    </Tooltip.Content>
+                  </Tooltip.Root>
+                )}
+              </div>
+              <RadioGroup.Root asChild defaultValue={account?.eventColor}>
+                <div className="flex pt-4 space-x-4">
+                  {colors?.map((color, ix) => {
+                    // const isSelected = account.color.id === id;
+
+                    return (
+                      <RadioGroup.Item
+                        key={ix}
+                        value={color}
+                        className="relative w-[2.25rem] h-[2.25rem] border rounded cursor-pointer"
+                        onClick={() =>
+                          updateEventColor({
+                            id: account.id,
+                            eventColor: color,
+                          })
+                        }
+                        style={{
+                          backgroundColor: themeColors[color][7],
+                          borderColor: themeColors[color][11],
+                        }}
+                      >
+                        <RadioGroup.Indicator asChild>
+                          <div className="absolute bottom-[2px] right-[2px] flex">
+                            <span className="inline-block w-3.5 h-3.5 -rotate-[140deg]">
+                              <div
+                                id="circle"
+                                className="absolute w-3.5 h-3.5 rounded-xl  left-[1px] top-[1px]"
+                                style={{
+                                  backgroundColor: themeColors[color][11],
+                                }}
+                              ></div>
+                              <div
+                                id="checkmark-stem"
+                                className={clsx(
+                                  [
+                                    "w-[1px] h-[8px] absolute top-[5px] left-1.5",
+                                  ],
+                                  `bg-${color}-9`,
+                                )}
+                                style={{
+                                  backgroundColor: themeColors[color][7],
+                                }}
+                              ></div>
+                              <div
+                                id="checkmark-kick"
+                                className={clsx([
+                                  "w-[5px] h-[1px] absolute top-[4px] left-1.5",
```

Comment:
hardcoded pixel values 🤔

Reply:
Yes, very hard to make [this answer](https://stackoverflow.com/a/21968532/5608461) work without it -- I tried initially but didn't look right. If you can, happy to adopt!